pkg/dns/provider/state.go

```diff
@@ -829,6 +829,10 @@ func (this *state) reconcileZone(logger logger.LogContext, zoneid string, entrie
 	modified := false
 	for _, e := range entries {
 		// TODO: err handling
+
```

Comment:
If the default handling is done at the central place above, it should be able to omit this second place